### PR TITLE
Add i18n Support

### PR DESCRIPTION
**Pull request by theduckylittle**
1. Support pluggable language
2. Attempt to translate most strings as they come in.
3. Support language testing.
4. Document how to add a language.

### CODE REVIEW

**Comment by theduckylittle** [comment]
Note: The included spanish translation is Google translate and it. is. AWFUL.

**Comment by theduckylittle** [comment]
To do before merging:

- [ ] Cleanup language JSON files
- [ ]  Take a whack at the Spanish translation (volunteers welcome)

**Comment by theduckylittle** [comment]
I took a manual approach to translating to spanish based on the QGIS translations file. I still needed to hand craft a few of the translations but I think they're good enough for someone to offer us feedback. :laughing:

File: src/gm3/components/catalog/tools/download.js [hunk 1]
```diff
@@ -75,68 +76,56 @@ function doDownload(features, downloadFormat) {
...
-}
-
-DownloadTool.defaultProps = {
-    helpText: 'Choose a format then click "Okay" to download layer features in that format.',
+                >
+                    <p>
+                        { t('download-help') }
+                    </p>
+                    <p>
+                        <label>Download format: </label>
```
Comment (klassenjs) [inline src/gm3/components/catalog/tools/download.js:114]: untranslated string

**Comment by klassenjs** [comment]
Also, Paper Size/Layout items in print diaglog

File: src/gm3/components/catalog/tools/download.js (hunk 1 above)
Comment (theduckylittle) [reply src/gm3/components/catalog/tools/download.js]: Got it

**Pull request merged**